src/date/format-properties.js

```diff
@@ -19,13 +20,28 @@ define([
  *
  * TODO Support other calendar types.
  */
-return function( pattern, cldr ) {
+return function( pattern, cldr, timeZone ) {
 	var properties = {
 			numberFormatters: {},
 			pattern: pattern,
 			timeSeparator: numberSymbol( "timeSeparator", cldr )
 		},
-		widths = [ "abbreviated", "wide", "narrow" ];
+		widths = [ "abbreviated", "wide", "narrow" ],
+		metaZone,
+		exemplarCity;
+
+		if ( timeZone ) {
+
+			// The latest metazone data of the metazone array.
+			//TODO expand to support the historic metazones based on the given date.
+			metaZone = cldr.supplemental([
+				"metaZones/metazoneInfo/timezone", timeZone, 0,
+				"usesMetazone/_mzone"
+			]),
```

Comment:
ok let me move this condition into https://github.com/globalizejs/globalize/pull/687/files#diff-a8a17cd910e21f0b9d3e8ffe4fcea2a8R50 to access `chr`